add information about gaslining (#119)
endless-sky/endless-sky-wiki: wiki/CreatingShips.md

* `"engine capacity"`: the amount of that outfit space which is suitable for installing engines. Some ships have lots of engine capacity but not much weapon capacity, or vice versa.

> * `"gaslining"`: not an engine attribute, but standardized use for landing on gas giants.

Reviewer: The other attributes in the Data section are specific to ships, so there isn't duplication with outfit data (like `"engine capacity"`), though `"gaslining"` is a bit unique as a custom attribute.

Should custom attributes like `"gaslining"` and `"spinal mount"` be added as bullets under Custom Attributes instead?
```suggestion
* `"gaslining"`: a [custom attribute](https://github.com/endless-sky/endless-sky/wiki/CreatingShips#custom-attributes), standardized to be used for landing on gas giants.
```
